### DISCUSSION TITLE
Prithwi Hegde: Resume Bundle X ATS Scoring

### DISCUSSION THREAD

**Discussion by Prithwi32** [recodehive/Stackoverflow-Analysis#354] [Q&A] [answered]
[Prithwi_Hegde_s_Resume_July.pdf](https://github.com/user-attachments/files/16325551/Prithwi_Hegde_s_Resume_July.pdf)
@sanjay-kv  sir this is my latest resume which I'm currently applying for salesforce and Commvault I would like know feedback on it

**Comment by SanyukthaPrabhu** [comment]
@Prithwi32 awesome resume!!
could u please give some suggestions as a senior year to me!

**Reply by Prithwi32** [reply]
> @Prithwi32 awesome resume!! could u please give some suggestions as a senior year to me!

@SanyukthaPrabhu sure, let me know for what specifically u want suggestions..
U can connect me in linkedin: https://www.linkedin.com/in/prithwi-hegde

**Comment by TatyOko28** [comment, marked as answer]
Hi Prithwi,
Your resume looks solid! 💪 Make sure to highlight Salesforce-related projects, certifications, and measurable achievements. Wishing you all the best in your job applications! 🚀

**Reply by Prithwi32** [reply]
> Hi Prithwi, Your resume looks solid! 💪 Make sure to highlight Salesforce-related projects, certifications, and measurable achievements. Wishing you all the best in your job applications! 🚀

Thank you..but i not got selected in those and also changed resume now

**Reply by TatyOko28** [reply]
Can you mark it as good answer, please?)
